react/leanShipping.js

```diff
@@ -294,35 +295,52 @@ export function getCombinedOption(
   )
 }
 
+// Get SLA's able to deliver all cart
+function getConsistentSlas(items) {
```

Comment:
I don't think I understood well the purpose of this function, can you clarify what it should do in a few words?

From what I understood, `items` is an array of array of SLAs (which are objects with an `id` property), and this function uses the SLAs from the first element in the `items` array and filters out the SLAs that don't exist on the remaining items of the `items` array, is that correct?

Reply:
I'm trying to think of another way to write this function to impose it's intent a bit clearer, but am failing miserably haha

Reply:
I think what you want is the intersection of all SLAs across all arrays inside items, right? I was a bit thrown off by the `if (consistentSlas.length === 0)` and thought that was something else I was missing, but that's just the initialization of  the array, but I don't know if it is correct (assuming you can have one item with no SLAs, and so this function should return an empty array)

Reply:
Yes. I was initializing. I think you're correct. I'll fix this too.

Reply:
I named it "consistent sla's" trying to explain that it appears consistently as an option in every item... but I think something like "slasAvailableToEveryItem" will be more explicit. hahaha